examples/quarkus-mvc/build.gradle

```diff
@@ -10,14 +11,17 @@ repositories {
 }
 
 dependencies {
-    implementation 'io.quarkus:quarkus-resteasy-jackson'
     implementation enforcedPlatform("${quarkusPlatformGroupId}:${quarkusPlatformArtifactId}:${quarkusPlatformVersion}")
     implementation 'io.quarkus:quarkus-arc'
     implementation 'io.quarkus:quarkus-qute'
     implementation 'io.quarkus:quarkus-resteasy-qute'
+    implementation 'io.quarkus:quarkus-opentelemetry'
 
-    // TL Java BE library
-    implementation 'com.truelayer:truelayer-java:4.0.0'
+    // Reference to the library version that's local to this project!
+    // Check out the main README for instructions on how to install
+    // the library in your project!
+    // https://github.com/TrueLayer/truelayer-java?tab=readme-ov-file#stable-releases
+    implementation 'com.truelayer:truelayer-java'
```

Comment:
We reference the local project, not via Maven. This should help us verifying builds at CI time!